Edits to commit scopes topic (#5116)
EnterpriseDB/docs: product_docs/docs/pgd/5/reference/commit-scopes.mdx

Example: `ANY 2 (left_dc)`

> A transaction under this commit scope group is considered committed if any two nodes in the `left_dc` group confirm they processed the transaction.

Reviewer: I feel like this is flattening out the temporal nature of the rule... "will be" is used because the rule in the scope is created before any transaction is processed. These rule elements are used in advance of transactions being processed and formulated by the reader.

Author: What if you changed "if" to "after"? Then you'd have the temporal nature, probably with more accuracy?

Reviewer: So "A transaction under this commit scope group will be considered committed after any two nodes in the...."

Sounds good to me.

Author: ```suggestion
A transaction under this commit scope group will be considered committed after any two nodes in the `left_dc` group confirm they processed the transaction.
```

Reviewer: ✔️

### ANY NOT

Example: `ANY 2 NOT (left_dc)`

> A transaction under this commit scope group is considered committed if any two nodes that aren't in the `left_dc` group confirm they processed the transaction.

Reviewer: ```suggestion
A transaction under this commit scope group will be considered committed if any two nodes that aren't in the `left_dc` group confirm they processed the transaction.
```

### MAJORITY

Example: `MAJORITY (left_dc)`

> A transaction under this commit scope group is considered committed if a majority of the nodes in the `left_dc` group confirm they processed the transaction.  

Reviewer: ```suggestion
A transaction under this commit scope group will be considered committed if a majority of the nodes in the `left_dc` group confirm they processed the transaction.  
```

### MAJORITY NOT

Example: `MAJORITY NOT (left_dc)`

> A transaction under this commit scope group is considered committed if a majority of the nodes that aren't in the `left_dc` group confirm they processed the transaction.

Reviewer: ```suggestion
A transaction under this commit scope group will be considered committed if a majority of the nodes that aren't in the `left_dc` group confirm they processed the transaction.
```

### ALL

Example: `ALL (left_dc)`

> A transaction under this commit scope group is considered committed if all of the nodes in the `left_dc` group confirm they processed the transaction.

Reviewer: ```suggestion
A transaction under this commit scope group will be considered committed if all of the nodes in the `left_dc` group confirm they processed the transaction.
```

### ALL NOT

Example: `ALL NOT (left_dc)`

> A transaction under this commit scope group is considered committed if all of the nodes that aren't in the `left_dc` group confirm they processed the transaction.

Reviewer: ```suggestion
A transaction under this commit scope group will be considered committed if all of the nodes that aren't in the `left_dc` group confirm they processed the transaction.
```
